### ISSUE TITLE
Show/link data collectors to the area they are assigned

### ISSUE THREAD

**Issue by smit1678** [hotosm/field-campaigner#306]
Send an email or link an editor directly to their task. Instead of just linking to the overall dashboard, link to a view of their area they need to cover.

Story: As a campaign manager, it would be helpful to send the individual team members doing the data collection a link (or email them a link) that shows the individual area that they need to collect. 

This wouldn't be a dashboard view but a task view of what needs to be collected and where.

**Comment by cchristelis** [comment]
@smit1678, is this a sub-area of a campaign?

**Comment by Dragneel7** [comment]
Hi @smit1678 , I would like to work on this issue.
Is it still open?

**Comment by smit1678** [comment]
@Dragneel7 Yes, this is still open. If you're interested in picking this up, great. Can you outline your approach here before you start?

**Comment by Dragneel7** [comment]
@smit1678 I am planning on implement this as part of GSoC project and will share with you a outline of my approach on gmail.
Hope that sounds good.

**Comment by Dragneel7** [comment]
@smit1678 I have added a workflow for this issue with the wireframes on how I aim to approach to this feature. Kindly review them.
link : [here](https://docs.google.com/document/d/1tLWl6WyY5A4TRK6NoaM4xYyOMkJcUn2oPM8OW5sacDY/edit?usp=sharing)
I have also added what I plan on implementing for GSoC, please review on them too and add if you think more features could be incorporated.
Thanks

**Comment by Dragneel7** [comment]
@smit1678 Here is my gsoc proposal in which I have addressed this issue. Your suggestions would be really helpful.
link : [here](https://docs.google.com/document/d/15Y1bQa1vc5p1h0IYfYk3_5IDw11gNKDiEJB-Sq8z73Q/edit)
